src/controllers/AdminController.php

```diff
@@ -334,12 +336,45 @@ class="fb-cta cta--yellow"
     $timer_on = $timer->getValue() === '1';
     $timer_off = $timer->getValue() === '0';
 
-    if ($start_ts->getValue() === '0') {
-      $start_ts = tr('Not started yet');
-      $end_ts = tr('Not started yet');
+    if ($start_ts->getValue() !== '0' && $start_ts->getValue() !== 'NaN') {
+      $game_start_ts = $start_ts->getValue();
```

Comment:
I'm not sure which section of code you thought would be better as an array.  I am likely just missing it.  Do you mind clarifying?